commons/typed-values/src/test/java/org/operaton/bpm/engine/test/variables/FileValueTypeImplTest.java

```diff
@@ -174,15 +178,17 @@ void createValueWithNullProperties() {
   @Test
   void cannotCreateFileWithoutName() {
     InputStream file = this.getClass().getClassLoader().getResourceAsStream("org/operaton/bpm/engine/test/variables/simpleFile.txt");
-    assertThrows(IllegalArgumentException.class, () ->
-      type.createValue(file, Collections.<String, Object>emptyMap()));
+    Map<String, Object> valueInfo = emptyMap();
```

Comment:
Maybe "emptyValueInfo" to highlight, that *this* is the variable which lets the test fail? Not a must, though, just an opinion

Reply:
Thx, changed that.